Pod/Classes/XExtensionItem.h

```diff
@@ -102,37 +96,23 @@ typedef UIImage *(^XExtensionItemThumbnailProvidingBlock)(CGSize suggestedSize);
  */
 @property (nonatomic, copy) NSDictionary *userInfo;
 
+- (instancetype)initWithURL:(NSURL *)URL;
```

Comment:
Yeah I wasn't sure how documentation should be handled for this so I just left it unfinished, sorry!